src/hello_app.py

```diff
@@ -1,13 +1,21 @@
 #this script set up the basic flask app
```

Comment:
we can lose this comment now.

also, can you rename the file to "main.py" and update the readme instructions accordingly please.

you'll also need to update the readme telling other developers how to run the tests.